src/amber/controller/helpers/responders.cr

```diff
@@ -13,30 +13,35 @@ module Amber::Controller::Helpers
       ACCEPT_SEPARATOR_REGEX = /,|,\s/
 
       @requested_responses : Array(String)
-      @available_responses = Hash(String, String).new
+      @available_responses = Hash(String, String | Proc(String) | Proc(Int32)).new
```

Comment:
Can we create an alias for the Proc type here to simplify. Have it scope to responders

```crystal
alias ProcType = Proc(String) | Proc(Int32)
```

Reply:
done